rotkehlchen/api/rest.py

```diff
@@ -1042,7 +1042,7 @@ def create_new_user(
             return {
                 'result': None,
                 'message': message,
-                'status_code': HTTPStatus.CONFLICT,
+                'status_code': HTTPStatus.UNAUTHORIZED,
```

Comment:
Shouldn't this stay a conflict contextually?

Reply:
I don't know. This is what I understood from what you asked. Just let me know

Reply:
401 makes sense for other endpoints but I still feel that for account creation or login 409 remains the valid response.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

> The HTTP 409 Conflict response status code indicates a request conflict with the current state of the target resource.

At least for me trying to login with a user while another user is still logged in feels like a conflict rather than being unauthorized.

Reply:
I have no opinions at all here. Before was fine for me, whatever you want changed is fine for me. So I will chanfge it back for accoutn creation and login

Reply:
So the way I have it in mind is that any "a user is not logged in" should be a 401 so that the frontend can have an interceptor that redirects you to the login screen. Does this clarify things?

Reply:
There is more 401s around the code for other things that match the unaothorized one. Not sure if it will always work. You can see if this is what you want, otherwise tell me which endpoint needs changing. Premium user only for example also get 401s.

The amount of manual changes in the docs make this a really annoying change to make. So please whatever you want to have specify very well beforehand, otherwise we have to pass over the docs again to make a lot of changes.